lib/eclipse/EclipseState/AquiferCT.cpp

```diff
@@ -0,0 +1,20 @@
+#include <opm/parser/eclipse/EclipseState/AquiferCT.hpp>
+
+namespace Opm {
+
+    AquiferCT::AquiferCT(const EclipseState& eclState, const Deck& deck)
+    {
+        m_aquifers = init_object(eclState, deck);
```

Comment:
I would prefer if the `init_objects` method was removed, this should just be the constructor.

Reply:
Can be done according to your review. It's up to @nairr how he wants to implement this. I made it as a skeleton/starting point.